src/App.js

```diff
@@ -1,38 +1,33 @@
-import React, { useState } from "react";
-import "./App.css";
+import React, { useState } from 'react';
+import './App.css';
```

Comment:
The main task of the reviewer of your code is to understand the changes that the commits/PR make. A reviewer typically has to put in a fair amount of time to understand what the code did before and how the code change modifies that behavior. A significant help to understanding the changes made are the red/green diff highlights. This cognitive aid along with general comprehension of code changes is hindered when there are unrelated diffs that need to be analyzed. In this case, the single and double quote changes make understanding the diff a more manual process:
![image](https://user-images.githubusercontent.com/5881537/106371247-bff04100-6327-11eb-9490-fd73926e5603.png)

Making a commit specific to formatting changes would help the reviewer remove this diff noise if the reviewer views the changes from the 'Commits' tab. Alternatively formatting changes could be done in an independent PR.

Reply:
Also, I noticed that in the PR you decided to change double quotes with single. The others team’s commit uses double quotes. You all should probably decide on a standard of which one you want to go with. I could see a scenario where you are changing others double quotes and they are changing your single quotes.